stream_alert_cli/test.py

```diff
@@ -332,15 +387,14 @@ def test_processor(self, alerts):
         Return:
             [bool] boolean indicating the status of the alert processor dispatching
         """
-        status = True
         # Set the logger level to info so its not too noisy
         StreamOutput.LOGGER.setLevel(logging.ERROR)
         for alert in alerts:
             if self.context.mocked:
                 self.setup_outputs(alert)
 
             for passed, output in StreamOutput.handler(alert, self.context):
-                status = status and passed
+                self.passing = self.passing and passed
```

Comment:
same comment - what's the difference between `passing` and `passed`? Readability is rough